src/crawler.ts

```diff
@@ -1477,29 +1479,28 @@ self.__bx_behaviors.selectMainBehavior();
     return false;
   }
 
-  awaitProcess(proc: ChildProcess) {
-    const stdout: string[] = [];
-    const stderr: string[] = [];
+  // todo: replace with js-wacz impl eventually
+  async _addCDXJ(wacz: WACZ) {
+    const dirPath = path.join(this.collDir, "tmp-cdx");
```

Comment:
if `this.params.generateCDX`is set, the CDX files may already be in `indexes` instead of `tmp-cdx` here